### PR TITLE
8294526: sun/security/provider/SubjectCodeSource.java no longer referenced

### PR DESCRIPTION
**Pull request by ryawalla**
SubjectCodeSource is no longer used, Class should be deleted and all references updated

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8294526](https://bugs.openjdk.org/browse/JDK-8294526): sun/security/provider/SubjectCodeSource.java no longer referenced

### Reviewers
 * [Weijun Wang](https://openjdk.org/census#weijun) (@wangweij - **Reviewer**) ⚠️ Review applies to [404856aa](https://git.openjdk.org/jdk/pull/10550/files/404856aacfb87b7ba0825f10949b1cf536dc8e19)
 * [Xue-Lei Andrew Fan](https://openjdk.org/census#xuelei) (@XueleiFan - **Reviewer**) ⚠️ Review applies to [64972419](https://git.openjdk.org/jdk/pull/10550/files/649724192dbb7eada27655d3b550760e745be55e)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/10550/head:pull/10550` \
`$ git checkout pull/10550`

Update a local copy of the PR: \
`$ git checkout pull/10550` \
`$ git pull https://git.openjdk.org/jdk pull/10550/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 10550`

View PR using the GUI difftool: \
`$ git pr show -t 10550`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/10550.diff">https://git.openjdk.org/jdk/pull/10550.diff</a>

</details>

### CODE REVIEW

File: test/jdk/sun/security/provider/PolicyFile/Comparator.java [hunk 1]
```diff
@@ -31,6 +31,9 @@
  * Note:  if you want to see the java.security.debug output,
  *        you can not simply set the system property.
  *        you must run this test by hand and pass -Djava.security.debug=...
+ *
+ * Note: SubjectCodeSource removed, but PolicyFile might have the same problem. Keep the test as-is
+ *
```
Comment (XueleiFan) [inline test/jdk/sun/security/provider/PolicyFile/Comparator.java:36]: I may have the note in a bug description instead.  Is it safe to remove this test and related files?

**Review by XueleiFan — CHANGES_REQUESTED** [review]

**Comment by seanjmullan** [comment]
@ryawalla Have you or your company signed the OCA? I do not see `/signed` or `/covered` actions in this PR.

**Comment by ryawalla** [comment]
@seanjmullan I thought I had resolved this as previously the OCA was failing jcheck, I am an Oracle employee.  Should I still be adding /signed or /covered actions? 
This is my first change in the openjdk so my account may not be linked to the org correctly.

**Comment by seanjmullan** [comment]
> @seanjmullan I thought I had resolved this as previously the OCA was failing jcheck, I am an Oracle employee. Should I still be adding /signed or /covered actions? This is my first change in the openjdk so my account may not be linked to the org correctly.

No, sorry for the false alarm, you should be all set then.

File: test/jdk/sun/security/provider/PolicyFile/Comparator.java (hunk 1 above)
Comment (ryawalla) [reply test/jdk/sun/security/provider/PolicyFile/Comparator.java:36]: @XueleiFan I have removed the Test and related policy files
(Comparator.Combined.Policy/ Comparator.Comparator.Policy/Comparator.Principal.Policy), is this what you were expecting? I can confirm I am still getting a successful test run.

Comment (XueleiFan) [reply test/jdk/sun/security/provider/PolicyFile/Comparator.java:36]: I meant to remove the Comparator.java.   As SubjectCodeSource has been removed, the note you added may be confusing for future readers.  If you want to have a record, you could use bug comment in JDK-8294526.

Comment (ryawalla) [reply test/jdk/sun/security/provider/PolicyFile/Comparator.java]: Thanks, I have removed the Comparator.java and related files and will add a comment on the bug

**Review by XueleiFan — APPROVED** [review]
Looks good to me.  Thanks!

**Comment by ryawalla** [comment]
@XueleiFan/ @wangweij Are you able to sponsor

**Pull request closed without merging**